### PR TITLE
Rename Microsoft.Dnx.Compilation.Abstractions to Microsoft.Extensions.Compilation.Abstractions 

### PR DESCRIPTION
**Pull request by pakrym**
Here go namespaces.

/cc @pranavkm  @davidfowl

### CODE REVIEW

File: test/Microsoft.Dnx.ApplicationHost.FunctionalTests/AppHostTests.cs [hunk 1]
```diff
@@ -20,10 +20,9 @@ public void LibraryExporterGetExports(DnxSdk sdk)
             sdk.Dnu.Restore(project.ProjectDirectory).EnsureSuccess();
 
             // Act
-            var result = sdk.Dnx.Execute(project);
+            var result = sdk.Dnx.Execute(project).EnsureSuccess();
```
Comment (pranavkm) [inline test/Microsoft.Dnx.ApplicationHost.FunctionalTests/AppHostTests.cs]: Why?

**Comment by pranavkm** [comment]
:shipit:

File: test/Microsoft.Dnx.ApplicationHost.FunctionalTests/AppHostTests.cs (hunk 1 above)
Comment (pakrym) [inline test/Microsoft.Dnx.ApplicationHost.FunctionalTests/AppHostTests.cs]: More info if it fails, EnsureSuccess dumps stderr

Comment (pranavkm) [inline test/Microsoft.Dnx.ApplicationHost.FunctionalTests/AppHostTests.cs]: Do the other tests need in this type need to be changed to do something similar?

**Pull request merged**